docs/data-ingestion/Alloy.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Grafana Alloy
+weight: 1
+sort: 1
+menu:
+  docs:
+    identifier: "alloy"
+    parent: "data-ingestion"
+    weight: 1
+aliases:
+  - /data-ingestion/Grafana-Alloy.html
+  - /data-ingestion/grafana-alloy.html
+  - /data-ingestion/Grafana-Agent.html
+  - /data-ingestion/grafana-agent.html
+
+---
+
+Grafana Alloy supports sending data via the Prometheus Remote Write Protocol and OpenTelemetry Protocol (OTLP).
+Collecting metrics and forwarding them to VictoriaMetrics using Prometheus scraping and remote write is more straight forward, but using OpenTelemetry enables more complex processing operations to occur before sending data to VictoriaMetrics.
+The Alloy configuration file can found in the following location depending on your platform
+
+- Linux: `/etc/alloy/config.alloy`
+- Windows: `%ProgramFiles%\GrafanaLabs\Alloy\config.alloy`
+- MacOS: `$(brew --prefix)/etc/alloy/config.alloy`
+
+After the configuration has been updated Alloy will need to be reloaded or restarted for the configuration change to be applied.
+
+- Linux: `sudo systemctl reload alloy.service`
+- Windows: `Restart-Service Alloy` This can also be done from the GUI using task manager
+- MacOS: `brew services restart alloy`
+- Helm chart: changing the `alloy.configMap` in the alloy [helm values](https://raw.githubusercontent.com/grafana/alloy/main/operations/helm/charts/alloy/values.yaml)
+
+In Any of the examples below you can add `insert/<tenant_id>/` to the URL path if you are sending metrics to vminsert.
+For Prometheus remote write this would change from
+
+```
+https://<victoriametrics-addr>/prometheus/api/v1/write
+```
+
+to
+
+```
+https://<vminsert-addr>/insert/<tenant_id>/prometheus/api/v1/write
+```
+
+For OpenTelemetry the endpoint would change from 
+
+```
+https://<victoriametrics-addr>:<victoriametrics_port>/opentelemetry
+```
+
+to
+
+```
+https://<vminsert-addr>:<victoriametrics_port>/insert/<tenant_id>/opentelemetry
+```
+
+## Collect Node Exporter data locally and send it to VictoriaMetrics without authentication
```

Comment:
All the text above doesn't explain which exact setting needs to be changed in alloy config. The first mention of the `prometheus.remote_write` happens only in this section, which has a bit unrelated title for a reader to follow. At least I perceive it as something additional, like "here's what you need to do if you scrape Node Exporter".

Reply:
I also think examples containing recommendations how to scrape something can be omitted. It is expected if someone uses alloy already, then they do the scraping already. 
So we can remove ` Collect Node Exporter ` or `Scrape Prometheus local Prometheus ` from titles and examples without losing useful info.

Reply:
To address this feedback I added a brief explanation of what needs to be added to alloy and that any prometheus scrape endpoint can be used we just used node exporter as an example.